docs/using-magnum-auto-healer.md

```diff
@@ -0,0 +1,227 @@
+    | uuid                                 | name                        | keypair         | node_count | master_count | status          |
+    +--------------------------------------+-----------------------------+-----------------+------------+--------------+-----------------+
+    | c418c335-0e52-42fc-bd68-baa8d264e072 | lingxian_por_test_1.12.7_ha | lingxian_laptop |          3 |            3 | CREATE_COMPLETE |
+    +--------------------------------------+-----------------------------+-----------------+------------+--------------+-----------------+
+    $ openstack server list --name lingxian-por-test-1-12-7-ha
+    +--------------------------------------+---------------------------------------------------+--------+-----------------------------------------+-------------------------+---------+
+    | ID                                   | Name                                              | Status | Networks                                | Image                   | Flavor  |
+    +--------------------------------------+---------------------------------------------------+--------+-----------------------------------------+-------------------------+---------+
+    | 908957c2-ac88-4b54-a1fc-91f9cc8f98f1 | lingxian-por-test-1-12-7-ha-bbgjts5g4xhb-minion-2 | ACTIVE | lingxian_net=10.0.10.33, 150.242.42.234 | fedora-atomic-27-x86_64 | c1.c4r8 |
+    | 8f0c3ad9-caf5-45b6-bf3a-97b3bb6de623 | lingxian-por-test-1-12-7-ha-bbgjts5g4xhb-minion-0 | ACTIVE | lingxian_net=10.0.10.32, 150.242.42.233 | fedora-atomic-27-x86_64 | c1.c4r8 |
+    | a6ae4cee-7cf2-4b25-89bc-a5c6cb2c364d | lingxian-por-test-1-12-7-ha-bbgjts5g4xhb-minion-1 | ACTIVE | lingxian_net=10.0.10.34, 150.242.42.245 | fedora-atomic-27-x86_64 | c1.c4r8 |
+    | 2af96203-cc6f-4b55-8fb2-062340207ebb | lingxian-por-test-1-12-7-ha-bbgjts5g4xhb-master-2 | ACTIVE | lingxian_net=10.0.10.31, 150.242.42.226 | fedora-atomic-27-x86_64 | c1.c2r4 |
+    | 10bef366-b5a8-4400-b2c3-82188ec06b13 | lingxian-por-test-1-12-7-ha-bbgjts5g4xhb-master-1 | ACTIVE | lingxian_net=10.0.10.30, 150.242.42.22  | fedora-atomic-27-x86_64 | c1.c2r4 |
+    | 9c17f034-6825-4e49-b3cb-0ecddd1a8dd8 | lingxian-por-test-1-12-7-ha-bbgjts5g4xhb-master-0 | ACTIVE | lingxian_net=10.0.10.29, 150.242.42.213 | fedora-atomic-27-x86_64 | c1.c2r4 |
+    +--------------------------------------+---------------------------------------------------+--------+-----------------------------------------+-------------------------+---------+
+    ```
+
+2. The kubeconfig file of the cluster is in place.
+
+### Deploy magnum-auto-healer
+
+It's recommended to run magnum-auto-healer service as a DaemonSet on the master nodes, the service is running in master-slave mode using leader election mechanism. There is a sample manifest file in `manifests/magnum-auto-healer/magnum-auto-healer.yaml`, you need to change some variables as need before actually running `kubectl apply` command. The following commands are just examples:
```

Comment:
Referring back to this previously reported issue[1], the use of the terminology "master-slave" is discouraged within the Kubernetes community, and the preferred nomenclature is "control-plane"[2] and "node"[3].

[1] https://github.com/kubernetes/website/issues/6525
[2] https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-control-plane
[3] https://kubernetes.io/docs/reference/glossary/?fundamental=true#node

Reply:
Changed to "active-passive"